src/node_crypto.cc

```diff
@@ -1974,7 +1974,89 @@ static Local<Object> X509ToObject(Environment* env, X509* cert) {
 }
 
 
-// TODO(indutny): Split it into multiple smaller functions
+static Local<Object> AddIssuerToObject(Local<Object> object,
+                                       X509** cert,
+                                       STACK_OF(X509)* const peer_certs,
+                                       Environment* const env) {
+  Local<Context> context = env->isolate()->GetCurrentContext();
+  *cert = sk_X509_delete(peer_certs, 0);
+  for (;;) {
+    int i;
+    for (i = 0; i < sk_X509_num(peer_certs); i++) {
+      X509* ca = sk_X509_value(peer_certs, i);
+      if (X509_check_issued(ca, *cert) != X509_V_OK)
+        continue;
+
+      Local<Object> ca_info = X509ToObject(env, ca);
+      object->Set(context, env->issuercert_string(), ca_info).ToChecked();
```

Comment:
I think we've settled on `.FromJust()` (232 vs 44 hits for `.ToChecked()` at the time of writing.)

Reply:
Thanks, I'll correct these.